### ISSUE TITLE
Filter of scan changed between Bleak 0.16.0 and 0.18.0

### ISSUE THREAD

**Issue by jeffsf** [jeffsf/pyDE1#35]
When starting to test under Python 3.11.0rc2 environment also moved from Bleak 0.16.0 to 0.18.0

Now seeing

```
  File "/home/pyde1/deploy/pyde1-devel/src/pyDE1/scale/processor.py", line 209, in change_scale_to_id
    await self.first_if_found(doit=True, warn_deprecated=False)
  File "/home/pyde1/deploy/pyde1-devel/src/pyDE1/scale/processor.py", line 195, in first_if_found
    device = await find_first_matching(
             ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pyde1/deploy/pyde1-devel/src/pyDE1/scanner.py", line 273, in find_first_matching
    return await BleakScannerWrapped.find_device_by_filter(filterfunc=is_match)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: BleakScanner.find_device_by_filter() missing 1 required positional argument: 'filter_func'
```

**Comment by jeffsf** [comment]
Fixed for bleak v0.18.0 which changed the signature of the method - 0b548b23
Subsequently removed patch with requirement for bleak v0.18.1 which restored original signature

**Comment by jeffsf** [comment]
Fixed in v1.5.0
